mobile-app/cypress/integration/functional/wallet/dex/compositeswap.spec.ts

```diff
@@ -126,6 +126,9 @@ context('Wallet - DEX - Composite Swap with balance', () => {
     cy.getByTestID('text_input_tokenA').type('10')
     cy.getByTestID('slippage_select').click()
     cy.getByTestID('slippage_1%').should('exist')
+    cy.url().should('include', 'app/DEX/CompositeSwap', () => {
```

Comment:
is the `cy.url()..` necessary? similarly for the two below

Reply:
As discussed, its a cypress command enqueued operation that is used with async storage usage. Will leave this for now.